tests/test_scss.py

```diff
@@ -2,19 +2,19 @@
 import pandas as pd
 
 from great_tables import GT
-from great_tables._scss import font_color, css_add, compile_scss
+from great_tables._scss import _font_color, _css_add, _compile_scss
 
 
 @pytest.mark.parametrize(
     "src,dst",
     [
-        ("#FFFFFF", "dark"),
```

Comment:
The changed `_font_color()` util function requires a valid color name in it's latter two args. So, I changed the param values of  `"light"` and `"dark"` to light and dark color hex values.